apps/voting/app/src/components/SummaryRows.js

```diff
@@ -1,17 +1,16 @@
 import React from 'react'
 import styled from 'styled-components'
-import { GU, textStyle, useTheme } from '@aragon/ui'
+import { GU, textStyle, formatTokenAmount, useTheme } from '@aragon/ui'
 import You from './You'
-import { formatNumber } from '../math-utils'
 import { VOTE_NAY, VOTE_YEA } from '../vote-types'
 
 function SummaryRows({ yea, nay, symbol, connectedAccountVote }) {
```

Comment:
Wanted to double check—what type is `yea` and `nay`? We're accessing `yea.pct` but I'm not sure this exists on the object anymore.